bot/src/keeper.ts

```diff
@@ -7,10 +7,25 @@ import {
     getCollateralAuthorizationStatus,
     getWalletAuthorizationStatus,
 } from 'auctions-core/src/authorizations';
-import { ETHEREUM_NETWORK, KEEPER_MINIMUM_NET_PROFIT_DAI, KEEPER_WALLET_PRIVATE_KEY } from './variables';
+import {
+    ETHEREUM_NETWORK,
+    KEEPER_MINIMUM_NET_PROFIT_DAI,
+    KEEPER_WALLET_PRIVATE_KEY,
+    KEEPER_WHITELIST,
+} from './variables';
 
 let isSetupCompleted = false;
 const currentlyExecutedAuctions = new Set();
+function isCollateralInWhitelist(collateralSymbol: string): boolean {
+    if (!KEEPER_WHITELIST) {
+        return true;
+    }
+    // First parse the string and make it into an array
+    const whiteListArray = KEEPER_WHITELIST.split(',');
```

Comment:
Given the string `"ETH-A, MATIC-A"` it will produce `" MATIC-A"` which will not be equal to `"MATIC-A"`

Reply:
Also, the env variable is parsed on every run of the function and not on startup as requested